pkgs/by-name/du/duplicity/package.nix

```diff
@@ -51,6 +51,10 @@ let
           --replace-fail /usr/bin /dev
       '';
 
+    pytestFlagsArray = [
```

Comment:
Can we put this in `disabledTests`?  It'll have to use the test name on its own (`test_black`).